src/microbe_stage/MicrobeHUD.cs

```diff
@@ -838,4 +838,38 @@ private void OnTranslationsChanged()
         UpdateColonySizeForMulticellular();
         UpdateColonySizeForMacroscopic();
     }
+
+    private void ToggleProcessPressed(ChemicalEquation equation)
+    {
+        if (!stage!.Player.Has<BioProcesses>())
```

Comment:
```suggestion
        if (!stage!.HasAlivePlayer || !stage.Player.Has<BioProcesses>())
```

Suggestion to more closely match other player related checks to use the general has alive player guard check.